src/App.js

```diff
@@ -1,23 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
+import LandingPage from './Containers/LandingPage/LandingPage';
```

Comment:
you can use a index file for most of the folders import statements become easier to manage.